hera_cal/vis_clean.py

```diff
@@ -1329,17 +1340,42 @@ def _clean_argparser():
 # ------------------------------------------
 
 
-def _linear_argparser():
+def _linear_argparser(parallelization_mode='file'):
     '''
     Arg parser for commandline operation of hera_cal.delay_filter in various linear modes.
 
     Returns
     -------
         Arg-parser for linear filtering. Still needs domain specific args (delay versus xtalk)
     '''
-    a = _filter_argparser()
+    a = _filter_argparser(parallelization_mode)
     cache_options = a.add_argument_group(title='Options for caching')
     a.add_argument("--write_cache", default=False, action="store_true", help="if True, writes newly computed filter matrices to cache.")
     a.add_argument("--cache_dir", type=str, default=None, help="directory to store cached filtering matrices in.")
     a.add_argument("--read_cache", default=False, action="store_true", help="If true, read in cache files in directory specified by cache_dir.")
     return a
+
+
+def _parse_baseline_list_string(baseline_list_string):
+    """
+    Helper method for parsing user provided lists of baselines.
+
+    Arguments
+    ---------
+        baseline_list_string: string
+            list of baselines provided as a string
+            that is a semi-colon separated list of antenna pairs.
+            e.g. 1, 2;0,0; 0, 10
+    Returns
+    -------
+        A list of baseline antenna pair tuples
+    """
+    # strip spaces
+    baseline_list_string = baseline_list_string.replace(" ", "")
```

Comment:
I'd suggest also cutting tabs and newlines. I like this way of handling it: https://stackoverflow.com/questions/3739909/how-to-strip-all-whitespace-from-string

Reply:
good idea. This has been done.